flask_appbuilder/security/api.py

```diff
@@ -95,14 +96,18 @@ def login(self):
         if not user:
             return self.response_401()
 
+        user_id = user.id
+        if isinstance(user_id, ObjectId):
```

Comment:
Can you make this more generic, without the MongoDB dependency at this level

Reply:
Okay, let me do it that way...